ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -216,7 +216,7 @@ private class JdbcLedgerDao(
       str("typ") ~
       str("submission_id") ~
       str("participant_id") ~
-      str("rejection_reason")(emptyStringToNullColumn).? ~
+      str("rejection_reason").map(s => if (s.isEmpty) null else s).? ~
```

Comment:
Maybe instead of `null` and `s` return `None` and `Some(s)` directly without using `.?`

Reply:
I think this was done so that both `null` and an empty string were evaluated to an empty value. Does it make sense?